frontend/src/app/components/wp-details/wp-details-toolbar.component.ts

```diff
@@ -29,7 +29,6 @@ import {WorkPackageEditingService} from '../wp-edit-form/work-package-editing-se
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {Component, Inject, Input} from '@angular/core';
 import {WorkPackageResource} from 'core-app/modules/hal/resources/work-package-resource';
-import {IWorkPackageEditingServiceToken} from "core-components/wp-edit-form/work-package-editing.service.interface";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/b6e4680ef80bc97abdec363477c560842ceb1115/frontend/src/app/components/wp-details/wp-details-toolbar.component.ts#L32"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>